python_modules/dagster/dagster/_core/storage/upath_io_manager.py

```diff
@@ -426,14 +426,6 @@ def load_input(self, context: InputContext) -> Union[Any, Dict[str, Any]]:
                 return self._load_multiple_inputs(context)
 
     def handle_output(self, context: OutputContext, obj: Any):
-        if context.dagster_type.typing_type == type(None):
```

Comment:
TODO - test that typing type Nothing with a non-None return value raises a type check error or some equivalent. Otherwise need to add that check in the parent PR